Add doc for fast tune (#10487)
pingcap/docs: grafana-tikv-dashboard.md

The Grafana dashboard is divided into a series of sub dashboards which include Overview, PD, TiDB, TiKV, Node\_exporter, Performance\_overview, and so on. A lot of metrics are there to help you diagnose.

## TiKV-Details dashboard

You can get an overview of the component TiKV status from the **TiKV-Details** dashboard, where the key metrics are displayed. According to the [Performance Map](https://asktug.com/_/tidb-performance-map/#/), you can check whether the status of the cluster is as expected.

> The following sections provide a detailed description of these key metrics on the **TiKV-Details** dashboard.

Reviewer: ```suggestion
This section provides a detailed description of these key metrics on the **TiKV-Details** dashboard.
```

    - raw_delete: The command of deleting a key/value pair
    - raw_batch_delete: The command of a batch of key/value pairs
    - raw_delete_range: The command of deleting a range of data

## TiKV-FastTune dashboard

> If performance issues of TiKV occur, such as QPS jitter, latency jitter, and latency increasing trend, you might need to check the **TiKV-FastTune** dashboard. This dashboard contains a set of panels and can help you with diagnostics, especially when the write workload of your cluster is medium or high.

Author: ```suggestion
If performance issues of TiKV occur, such as QPS jitter, latency jitter, and latency increasing trend, you might need to check the **TiKV-FastTune** dashboard. This dashboard contains a set of panels and can help you with diagnostics, especially when the write workload of your cluster is medium or large.
```

> If performance issues of TiKV occur, such as QPS jitter, latency jitter, and latency increasing trend, you might need to check the **TiKV-FastTune** dashboard. This dashboard contains a set of panels and can help you with diagnostics, especially when the write workload of your cluster is medium or large.

Reviewer: ```suggestion
If performance issues of TiKV occur, such as QPS jitter, latency jitter, and latency increasing trend, you can check the **TiKV-FastTune** dashboard. This dashboard contains a set of panels that help you with diagnostics, especially when the write workload in your cluster is medium or large.
```

If performance issues of TiKV occur, such as QPS jitter, latency jitter, and latency increasing trend, you can check the **TiKV-FastTune** dashboard. This dashboard contains a set of panels that help you with diagnostics, especially when the write workload in your cluster is medium or large.

When write-related performance issues occur, you can first check the TiDB dashboard, and **TiKV latency** panel of the **TiDB Summary** dashboard in Grafana. If the issues are at the storage side, open the **TiKV-FastTune** page, browse and check every panel on it.

In the **TiKV-FastTune** dashboard, you can see a title that suggests a possible cause of the performance issues. To check whether the suggested cause is true, check the graph on the page.

The left-Y-axis of the graph represents the write-RPC QPS of the storage side, and a set of graphs on the right-Y-axis are drawn upside down. If the shape of the left graph matches that of the right graphs, the suggested cause is true.

For detailed metrics and descriptions, see the dashboard [user manual](https://docs.google.com/presentation/d/1aeBF2VCKf7eo4-3TMyP7oPzFWIih6UBA53UI8YQASCQ/edit#slide=id.gab6b984c2a_1_352).

> 

Author: ```suggestion
```
